First draft of ThunderHub chapter for myNodeDocs (#20)
mynodebtc/mynode_docs: docs/lightning/thunderhub.md

# Thunderhub

A GUI to manage lightning channels

### Introduction

ThunderHub is an open-source LND node manager where you can manage and monitor your node on any device or browser. It allows you to take control of the lightning network with a simple and intuitive UX.

ThunderHub is a creation of Anthony Potdevin, and complete information can be found at either [Github](https://github.com/apotdevin/thunderhub) or the [ThunderHub
website](https://thunderhub.io).

> ![ThunderHub GUI](/docs/.vuepress/public/images/lightning/ThunderHub/ThunderHub_GUI_comp.png "ThunderHub GUI")

Reviewer: Instead for the full address, you can just use "/images/lightning/ThunderHub/ThunderHub_GUI_comp.png"

Author: OK, thanks. I wasn't sure. When writing for the Wasabi docs, all you would need is "/imagename.png".
I wasn't sure that would work with our setup, so I opted for the full path.
Regardless, when I use yarn to view my page on my browser, the images do not render. Would I need to keep the image reference in my page to a local address for that to work, then change the address before pushing the page?

Reviewer: Yarn should work with my suggested address format, so something else might be wrong

Author: Yes, it is working fine now.

### Features

- Monitoring
- Overview of current and pending balance for the Lightning and Bitcoin wallets.
- Liquidity report with total remote and local lightning balance.
- Complete network info.
- View open/pending/closed channels and how balanced they are.
- View all transactions.
- View all forwarded payments.
- View all chain transactions.
- View all unspent UTXOS.

### Management

- Send and Receive Lightning payments.
- Send and Receive Bitcoin payments.
- Open and close channels.
- Balance your channels through circular payments. (Check out the Tutorial)
- Backup, verify and recover all your channels.
- Sign and verify messages.

## ThunderHub on myNode

To use ThunderHub on myNode, open the main myNode page on a browser and find the ThunderHub emblem.

![ThunderHub Emblem](/docs/.vuepress/public/images/lightning/ThunderHub/ThunderHub_emblem_comp.png "ThunderHub Emblem")

> 

Reviewer: probably you wanted to have some other image here, but this is same as the last one

Click on the "Enable" button, and a new tab should open on your browser, containing ThunderHub.
Go to that tab, enter your myNode password where requested, and then press

> "Connect".

Reviewer: Add few more things in this subsection
- mention the port number(s) where thunderhub opens
- any limitations with myNode implementation, I can't think of any right now
- location of installation inside myNode: `/opt/mynode/thunderhub`

Author: Those are good suggestions. I'm on the road today, but will try to implement for further review before I leave.
I'll check the images also. They are indeed supposed to be different.

Author: Do you think the length is OK? If anything, it could have more detail on how to use, but that leads to a potential for a much longer page. I'm OK with that, but curious as to your "vision" of how the docs should function in that regard.

Reviewer: Don't make it too long with step by step guide. That should be the app developer's responsibility. Instead, provide links to well written articles or videos which do so. That way, we don't have to reinvent the wheel.

Author: Perfect. I totally agree

The ThunderHub installation on myNode is located at: /opt/mynode/thunderhub.

![ThunderHub GUI](/images/lightning/ThunderHub/ThunderHub_GUI_comp.png "ThunderHub GUI")

### Features

- Monitoring
- Overview of current and pending balance for the Lightning and Bitcoin wallets.
- Liquidity report with total remote and local lightning balance.
- Complete network info.
- View open/pending/closed channels and how balanced they are.
- View all transactions.
- View all forwarded payments.
- View all chain transactions.
- View all unspent UTXOS.

### Management

- Send and Receive Lightning payments.
- Send and Receive Bitcoin payments.
- Open and close channels.
- Balance your channels through circular payments. (Check out the Tutorial)
- Backup, verify and recover all your channels.
- Sign and verify messages.

## ThunderHub on myNode

To use ThunderHub on myNode, open the main myNode page on a browser and find the ThunderHub emblem.

![ThunderHub Emblem](/images/lightning/ThunderHub/ThunderHub_emblem_comp.png "ThunderHub Emblem")

> Click on the "Enable" button, and a new tab should open on your browser, containing ThunderHub.

Author: ```suggestion
Click on the "Enable" button, and wait for the page to refresh. You should now see a "ThunderHub" button where the "Enable" button was. Click this button and a new tab should open on your browser, containing ThunderHub.
```

> Go to that tab, enter your myNode password where requested, and then press

Author: ```suggestion
Go to that tab, click the "Connect" button and enter your myNode password where requested, and then click on "Connect" again.
```

> "Connect".

Author: ```suggestion

```
